python/tank/bootstrap/bundle_downloader.py

```diff
@@ -35,10 +35,7 @@ def __init__(self, connection, pipeline_config_id, descriptor):
         base_class_path = os.path.normpath(
             os.path.join(
                 os.path.dirname(__file__),  # ./python/tank/bootstrap
-                "..",  # ./python/tank
-                "..",  # ./python
-                "..",  # ./
-                "hooks",  # ./hooks
+                "hooks",
                 "bootstrap.py",  # ./hooks/bootstrap.py
```

Comment:
I think the paths described in the comments are confusing. The final path should look something like 
`.../tk-core/python/tank/bootstrap/hooks/bootstrap.py`